Makefile

```diff
@@ -164,15 +164,15 @@ image:
 # Run local e2e smoke test against the checked-out code
 # using a local kind cluster.
 ###############################################################################
-E2E_FOCUS ?= "sig-network.*Conformance|sig-calico.*Conformance"
+E2E_FOCUS ?= "sig-network.*Conformance|sig-calico.*Conformance|BGP"
```

Comment:
Any reason to have the new BGP tests be in the default focus? Were the old python tests run as part of conformance?

Reply:
This isn't really meant to define what Conformance is, it's just meant to define what tests we run as part of CI.

I should probably rename the Semaphore Job.